pkg/restapi/vc/operation/operations_test.go

```diff
@@ -345,7 +338,7 @@ func TestNew(t *testing.T) {
 	t.Run("test error from open store", func(t *testing.T) {
 		client := edv.NewMockEDVClient("test", nil)
 		op, err := New(&mockstore.Provider{ErrOpenStoreHandle: fmt.Errorf("error open store")},
-			client, &kmsmock.CloseableKMS{}, &vdrimock.MockVDRIRegistry{}, "localhost:8080")
+			client, getTestKMS(t), &vdrimock.MockVDRIRegistry{}, "localhost:8080")
```

Comment:
I noticed that we were using a mock KMS in a lot of places that didn't need a mock - using a real one works fine. Hence you'll see a bunch of these mock KMS calls replaced with a real one.